src/ai/defaultai.cc

```diff
@@ -4260,6 +4262,8 @@ bool DefaultAI::check_productionsites(uint32_t gametime) {
 	// Get link to productionsite that should be checked
 	ProductionSiteObserver& site = productionsites.front();
 
+
+
```

Comment:
Any reason for these 2 empty lines?

Reply:
no defintily no reason maybe a leftover of some logs.

Reply:
funny thing is according to the log the script was already run by travis. The complaint in the log was about productionsites.cc where nothing has changed except 2 values in a formula.

Reply:
Travis runs the script only for testing but doesn't actually change anything. So, when it fails on the "Code Formatting" test it means you need to run fix_formatting.py locally and push the changes to the branch. On Launchpad bunny-bot took care of the formatting.

Reply:
how do i do this on windows?

Reply:
Go to the base folder of your widelands git-repo.

Run ```python utils/fix_formatting.py```from there.

Reply:
also consider installing provided git hooks which will take care of this automatically when comitting/pushing (execute <widelands-repo>/install-githooks.sh

Reply:
had a look into the script for c++ clang-format is called which I don't have on windows. Travis should change the files imho. perhaps something went wrong with the git hook of travis.

Reply:
merged the format fix branch so I believe this resolved